src/harness/testcases/WINNF_FT_S_SCS_testcase.py

```diff
@@ -64,3 +65,123 @@ def test_WINNF_FT_S_SCS_5(self):
+
+    Checks that SAS UUT response with fatal alert with unknown_ca.
+    """
+    config = loadConfig(config_filename)
+    self.assertTlsHandshakeFailure(client_cert=config['clientCert'],
+                                   client_key=config['clientKey'])
+
+  def generate_SCS_7_default_config(self, filename):
+    """Generates the WinnForum configuration for SCS_7"""
+    # Create the actual config for client cert/key path
+
+    config = {
+      'clientCert': self.getCertFilename("corrupted_client.cert"),
+      'clientKey': self.getCertFilename("corrupted_client.key")
+    }
+    writeConfig(filename, config)
+
+  @configurable_testcase(generate_SCS_7_default_config)
+  def test_WINNF_FT_S_SCS_7(self,config_filename):
+    """Corrupted certificate presented during registration.
+
+    Checks that SAS UUT response with fatal alert message.
+    """
+    config = loadConfig(config_filename)
+    self.assertTlsHandshakeFailure(client_cert=config['clientCert'],
+                                   client_key=config['clientKey'])
+
+  def generate_SCS_8_default_config(self, filename):
+    """Generates the WinnForum configuration for SCS_8"""
+    # Create the actual config for client cert/key path
+
+    config = {
+      'clientCert': self.getCertFilename("self_signed_client.cert"),
+      'clientKey': self.getCertFilename("client.key")
+    }
+    writeConfig(filename, config)
+
+  @configurable_testcase(generate_SCS_8_default_config)
+  def test_WINNF_FT_S_SCS_8(self,config_filename):
+    """Self-signed certificate presented during registration.
+
+    Checks that SAS UUT response with fatal alert message.
+    """
+    config = loadConfig(config_filename)
+    self.assertTlsHandshakeFailure(client_cert=config['clientCert'],
+                                   client_key=config['clientKey'])
+
+  def generate_SCS_9_default_config(self, filename):
+    """Generates the WinnForum configuration for SCS_9"""
+    # Create the actual config for client cert/key path
+
+    config = {
+      'clientCert': self.getCertFilename("non_cbrs_signed_device.cert"),
+      'clientKey': self.getCertFilename("non_cbrs_signed_device.key")
+    }
+    writeConfig(filename, config)
+
+  @configurable_testcase(generate_SCS_9_default_config)
+  def test_WINNF_FT_S_SCS_9(self,config_filename):
+    """Non-CBRS trust root signed certificate presented during registration.
+
+    Checks that SAS UUT response with fatal alert message.
+    """
+    config = loadConfig(config_filename)
+    self.assertTlsHandshakeFailure(client_cert=config['clientCert'],
+                                   client_key=config['clientKey'])
+
+  def generate_SCS_10_default_config(self, filename):
+    """Generates the WinnForum configuration for SCS_10. """
+    # Create the actual config for client cert/key path
+
+    # Load Devices
+    device_a = json.load(
+      open(os.path.join('testcases', 'testdata', 'device_a.json')))
+
+    # Create the actual config.
+    devices = [device_a]
```

Comment:
Why loading this device_a that will be not used in config?

Reply:
fixed. thanks.